
---
float64 elapsed_time
---
geometry_msgs/Vector3 contact_point_estimate
geometry_msgs/Vector3 contact_point_computed
float64 elapsed_time
